api/migrations/0024_remove_civi_links.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0023_auto_20170615_0827'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
@jl24 please review this migration. When I started working with my Bill model, and ran `./manage.py makemigrations`, this migration was created. I haven't worked with the Civi model code, so this may have been removed in a different pull request.

Reply:
This is ok. I believe it was neglected last time during deployment to the dev servers.